src/views/v2/tickets/item.php

```diff
@@ -66,18 +66,21 @@
 	<?php endif; ?>
 >
 
-	<?php $this->template( 'v2/modal/item/remove', $context ); ?>
+	// @todo Convert this into an action.
+	<?php //$this->template( 'v2/modal/item/remove', $context ); ?>
```

Comment:
* :warning: This comment is 67% valid code; is this commented out code?
* :no_entry_sign: No space found before comment text; expected "// $this->template( 'v2/modal/item/remove', $context );" but found "//$this->template( 'v2/modal/item/remove', $context );"

<sub>hash: 8842cbc58aa3902a2173f98e88885859</sub>